docs/extensions.md

```diff
@@ -0,0 +1,23 @@
+# Extensions
+
```

Comment:
Extensions mechanism allows creating complex user scenarios, especially in case of customized setups or permissioned networks.

Extension developer's guide:
1. Create new NetCore 3.1 library project in VisualStudio. 
2. Add references to `Centaurus.Domain` and `Centaurus.Common` projects.
3. Implement `IExtension` interface from `Centaurus.Domain` project. 
4. Subscribe to one or more events of `Global.ExtensionsManager` instance.
5. Create an extension configuration file and place the extension dll file to the same folder.
6. Specify the path to the extension config file using `--extensions_config_file_path path-to-config` param while starting Alpha and auditors.

Where is the description of the extension config file format?
Which extension points are supported? Where is a doc for them (description, arguments)?

And please spend time on docs formatting. Quality of docs is more important than the code itself in this case.